src/ActionModal/components/DelegationModal.vue

```diff
@@ -358,17 +358,7 @@ export default {
   validations() {
     return {
       amount: {
-        required: (x) => {
-          if (
-            (this.currentNetwork.network_type === "polkadot" &&
-              this.totalStaked > 0) ||
-            this.session.addressRole === `controller`
-          ) {
-            return true
-          } else {
-            return !!x && x !== `0`
-          }
-        },
+        required: (x) => (!!x && x !== `0`) || this.session.addressRole === `controller`,
```

Comment:
nice simplification, but could be confusing in the future

Reply:
Why? Can we improve somehow?

Reply:
BTW: I tested using stash, controller and stash/controller in polkadot and stake/restake in cosmos just to make sure it works ^^

Reply:
just mentioning, because addressRole is somehow implicit to polkadot.

Reply:
Yep, true! the good thing is that we can safely skip it for other network types (I hope ^^)